cmd/daytona/config/autocompletion.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"os"
+	"runtime"
+	"strings"
+)
+
+const completionScriptNameRoot = "daytona.completion_script."
+
+func DeleteAutocompletionData() error {
+	shellName := getCurrentShell()
```

Comment:
This should be done for all shells because the user chooses a shell when adding autocomplete. It's not guaranteed to be the users current shell.